app/Http/Controllers/LabelsController.php

```diff
@@ -65,6 +66,17 @@ public function show(string $labelName)
         $exampleAsset->model->category->id = 999999;
         $exampleAsset->model->category->name = trans('admin/labels/table.example_category');
 
+        $customFieldColumns = CustomField::all()->pluck('db_column');
+
+        collect(explode(';', Setting::getSettings()->label2_fields))
+            ->each(function ($item) use ($customFieldColumns, $exampleAsset) {
+                $pair = explode('=', $item);
+
+                if ($customFieldColumns->contains($pair[1])) {
```

Comment:
I led you astray 😬 

In my database my `settings.label2_fields` was
```
name=name;serial=serial;model=model.name;
```

That last `;` resulted in an empty element after the `collect(explode(';', Setting::getSettings()->label2_fields))` line:
```
  0 => "name=name"
  1 => "serial=serial"
  2 => "model=model.name"
  3 => ""
```

So trying to access `$pair[1]` caused an error.

Adding a `filter()` seems like a good fix:
```php
collect(explode(';', Setting::getSettings()->label2_fields))
            ->filter()
            ->each(function ($item) use ($customFieldColumns, $exampleAsset) {
                $pair = explode('=', $item);

                if ($customFieldColumns->contains($pair[1])) {
                    $exampleAsset->{$pair[1]} = "{{$pair[0]}}";
                }
            });
```

That will result in the last (or any) `""`s being removed before actually processing the items.